dotnet/hello_world/Program.cs

```diff
@@ -0,0 +1,210 @@
+﻿using Google.Cloud.Bigtable.Admin.V2;
+using Google.Cloud.Bigtable.V2;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Google.Protobuf;
+using System.Linq;
+using System.IO;
+using System;
+
+
+namespace HelloWorld
+{
+    class Program
+    {
+        // Connect to bigtable and perform basic operations
+        // 1. Create table with column family
+        // 2. Insert rows
+        // 3. Read rows
+        // 4. Delete table
+        //String[] x = Environment.GetCommandLineArgs();
+
+        static HelloWorldSettings _settings;
+
+        static void Main(string[] args)
+        {
+            #region Setting up application settigns
+            var builder = new ConfigurationBuilder().
+                SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);
+
+            IConfigurationRoot configuration = builder.Build();
+
+            _settings = new HelloWorldSettings();
+            configuration.Bind(_settings);
+            #endregion
+
+            HelloWorld helloWorld = new HelloWorld(_settings.ProjectId, _settings.InstanceId);
+            helloWorld.DoHelloWorld();
+            Console.WriteLine("Press any key to exit");
+            Console.ReadKey();
+        }
+
+        public class HelloWorld
+        {
+
+            public string ProjectId { get; set; }
+            public string InstanceId { get; set; }
+
+            public HelloWorld(string projectId, string instanceId)
+            {
+                ProjectId = projectId;
+                InstanceId = instanceId;
+            }
+
+            public void DoHelloWorld()
+            {
+
+                // Build Instance Name
+                InstanceName instanceName = new InstanceName(ProjectId, InstanceId);
+
+                // Build table name Admin
+                Google.Cloud.Bigtable.Admin.V2.TableName _tableAdmin = new Google.Cloud.Bigtable.Admin.V2.TableName(ProjectId, InstanceId, _settings.TableName);
+
+                // Build table name Client
+                Google.Cloud.Bigtable.V2.TableName _tableClient = new Google.Cloud.Bigtable.V2.TableName(ProjectId, InstanceId, _settings.TableName);
+
+                #region 1. Create table with column family
+                // Use try-with-resources to make sure it gets closed
+                try
+                {
+                    BigtableTableAdminClient bigtableTableAdminClient = BigtableTableAdminClient.Create();
+
+                    Console.WriteLine($"Creating new table: {_settings.TableName}");
+                    // Checking if a table with given tableName already exists
+                    var tables = bigtableTableAdminClient.ListTables(instanceName);
+
+                    bool isTableExist = tables.Any(x => x.TableName.TableId == "HelloBigTable");
```

Comment:
s/BigTable/Bigtable/g — Bigtable is not camel-cased; please make this change globally.